<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find dekf)/config/my_arg_file.yaml"/>-->
 
 <node pkg="dekf" type="dekf_node" name="DekfNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find dekf)/config/config_dekf.yaml"/>
</node>

</launch>
